packages/paste-website/src/pages/patterns/pricing/index.mdx

```diff
@@ -0,0 +1,390 @@
+              Package includes:
+            </Heading>
+            {[
+              "10 seats",
+              "2 sources",
+              "Unlimited destinations",
+              "1,000 MTU",
+              "50 Function hours",
+              "500,000 records processed for Reverse ETL",
+            ].map((item: string) => (
+              <Box key={item} display="flex" columnGap="space20" alignItems="center">
+                <SelectedIcon decorative color="colorTextIconSuccess" />
+                <Paragraph marginBottom="space0">{item}</Paragraph>
+              </Box>
+            ))}
+          </Box>
+        </Box>
+      </Card>
+      {...}
+    </Box>
+  )
+}`}
+/>
+
+### Visual picker
+
+Use a visual picker when the selection is happening in a wizard and the user will be continuing on to additional steps besides a simple confirmation (ex. Payment information).
+The visual picker should be more succinct in the detail it shares.
+
+<StoryPreview
+  title="Visual picker example"
+  storyID="website-pricingexamples--visual-picker-example"
+  height="600px"
+  code={`
+export const VisualPickerExample = (): React.ReactNode => {
+  const [value, setValue] = React.useState('Free trial');
+  return (
+    <VisualPickerRadioGroup
+      orientation="horizontal"
+      legend="Select an option"
+      name="visual-picker"
+      value={value}
+      onChange={(newValue) => setValue(newValue)}
+    >
+      <VisualPickerRadio labelText="Free trial value="Free trial>
+        <Box>
+          <Box display="flex" columnGap="space30" justifyContent="space-between" alignItems="flex-start">
+            <Heading as="h2" variant="heading40" marginBottom="space0">
+              Free trial
+            </Heading>
+            <Badge as="span" variant="success">
+              Recommended
+            </Badge>
+          </Box>
+          <Box marginTop="space30" display="flex" columnGap="space20" alignItems="flex-end">
+            <Heading as="h3" variant="heading10" marginBottom="space0">
+              $0
+            </Heading>
+            <Paragraph marginBottom="space0">Per month</Paragraph>
+          </Box>
+        </Box>
+        <Box marginY="space50">
+          <Separator orientation="horizontal" />
+        </Box>
+        <Box display="flex" flexDirection="column" rowGap="space50">
+          {["$15 credit", "Testing with upto 5 verified recipients", "Limited configurations"].map((item: string) => (
+            <Box key={item} display="flex" columnGap="space20" alignItems="center">
+              <SelectedIcon decorative color="colorTextIconSuccess" />
+              <Paragraph marginBottom="space0">{item}</Paragraph>
+            </Box>
+          ))}
+        </Box>
+      </VisualPickerRadio>
+      {...}
+    </VisualPickerRadioGroup>
+  )
+}`}
+/>
+
+### Table
+
+Consider using a table when there are 5 or more options for users to choose from, or more comparison is necessary between any number of plans.
+
+<StoryPreview
```

Comment:
chore: Use a Data Grid for this example since there are meant to be interactive elements (Buttons)